app/assets/javascripts/modules/sticky-element-container.js

```diff
@@ -7,28 +7,28 @@
   Use 'data-module="sticky-element-container"' to instantiate, and add
   `[data-sticky-element]` to the child you want to position.
 */
-(function (Modules, root) {
+(function (Modules, window) {
```

Comment:
Nice renaming. One thought - since we're passing `window` into this code and referring to it as `window`, do we need to pass it anymore? It should be globally accessible from anywhere, right?

Reply:
Ah yes, absolutely correct - thanks for picking that up 👍 Fixed up in last commit.